products/views.py

```diff
@@ -1,3 +1,48 @@
-from django.shortcuts import render
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q
 
-# Create your views here.
+from products.models  import Product, ProductSize
+
+class ProductsView(View):
+    def get(self, request): 
+        try:
+            ordering = request.GET.get('ordering', None)
+            sort     = request.GET.get('sort', "id")
+            size     = request.GET.get('size', None).split(',')
+            page     = int(request.GET.get('page', 0))
```

Comment:
```suggestion
            offset     = int(request.GET.get('page', 0))
```
아래 로직을 보니 offset이 더 맞는 의미인 것 같네요.

Reply:
네 확인하였습니다!